modules/installation-custom-aws-vpc.adoc

```diff
@@ -4,11 +4,15 @@
 // * installing/installing_aws/installing-aws-private.adoc
 // * installing/installing_aws/installing-aws-vpc.adoc
 
-[id="installation-custom-aws-vpc_{context}"]
-= About using a custom VPC
 ifeval::["{context}" == "installing-aws-china-region"]
 :aws-china:
 endif::[]
+ifeval::["{context}" == "installing-aws-vpc"]
+:public:
+endif::[]
+
+[id="installation-custom-aws-vpc_{context}"]
+= About using a custom VPC
 
 In {product-title} {product-version}, you can deploy a cluster into existing subnets in an existing Amazon Virtual Private Cloud (VPC) in Amazon Web Services (AWS). By deploying {product-title} into an existing AWS VPC, you might be able to avoid limit constraints in new accounts or more easily abide by the operational constraints that your company's guidelines set. If you cannot obtain the infrastructure creation permissions that are required to create the VPC yourself, use this installation option.
 
```

Comment:
Optional (especially since this is existing content), I'd suggest spelling out CIDR on first reference:
Classless inter-domain routing (CIDR)

Reply:
@bmcelvee While I agree that it is best practice to spell out acronyms on first reference, CIDR is a networking term that has become ubiquitous. I believe this is a case where spelling it out is not required. WDYT?

Reply:
I tend to spell them out regardless, but it's just a personal preference. I think it's okay either way here. :)